src/commands/init.js

```diff
@@ -46,6 +46,16 @@ exports.handler = function ({ reporter, name, template }) {
   
   const basename = name.split('.')[0]
   const tasks = new TaskList([
+    {
+      title: 'Checking project existence',
+      task: async (ctx, task) => {
+        const projectPath = path.resolve(process.cwd(), basename)
+        const exists = await fs.pathExists(projectPath)
+        if (exists) {
+          throw new Error(`Couldn't initialize project. Project with name ${basename} already exists in ${projectPath}. Use different <name> or rename existing project folder.`)
```

Comment:
First stab on meaningful error message. Happy to add better wording here if anyone has strong opinions :)

Reply:
LGTM!